apm-webapp/src/main/resources/application.yml

```diff
@@ -17,28 +17,28 @@
 server:
   port: 8080
 
-zuul:
-  ignoredServices: '*'
-  routes:
-    api:
-      path: /graphql
-      serviceId: collector
-    login:
-      path: /login/account
-      serviceId: collector
-
-collector:
-  path: /graphql
-  ribbon:
-    # Point to all backend's restHost:restPort, split by ,
-    listOfServers: 127.0.0.1:12800
-
 spring:
-  resources:
-    add-mappings: false
+  cloud:
+    gateway:
+      routes:
+        - id: oap-route
+          uri: lb://oap-service
```

Comment:
Where does this mean?

Reply:
`lb:` represents client-side  `load balance`